galeracluster/source/versioninginformation.rst

```diff
@@ -3,68 +3,64 @@
 ================================
 .. _`Versioning Information`:
 
-Galera Cluster consists of two software packages:
 
-- Galera wsrep Provider
-- MySQL server patched with the wsrep :abbr:`API (Application Programming Interface)`
 
-This chapter describes the Galera Cluster release numbering schemes and software packages.
+Galera Cluster for MySQL is available in binary software packages for several different Linux distributions, as well as in source code for other distributions and other Unix-like operating systems, such as FreeBSD and Solaris. 
+
+For Linux distributions, binary packages in 32-bit and 64-bit for both the MySQL database server with the wsrep API patch and the :term:`Galera Replication Plugin` are available from the `Codership Repository <releases.galeracluster.com>`_.  These include support for:
+
+- Red Hat Enterprise Linux
+- Fedora
+- CentOS
+- SUSE Linux Enterprise Server
+- openSUSE
+- Debian
+- Ubuntu
+
+By installing and configuring the Codership Repository on any of these systems, you can install and update Galera Cluster for MySQL through your package manager.  In the event that you use a distribution of Linux that is not supported, or if you use another Unix-like operating system, source files are available on GitHub, at:
+
+- `MySQL Server <https://github.com/codership/mysql-wsrep>`_ with the wsrep API patch.
+- `Galera Replication Plugin <https://github.com/codership/galera>`_.
+- `wsrep API <https://github.com/codership/wsrep-API>`_, alone, for patching MySQL Server yourself.

```

Comment:
Lets not mention the wsrep-API repository in here, as it is a very advanced concept with very limited usefulness to normal people